lib/business_calendar/version.rb

```diff
@@ -1,3 +1,3 @@
 module BusinessCalendar
-  VERSION = "2.0.0"
+  VERSION = "2.0.2"
```

Comment:
you didn't actually push the incorrect `v2.0.1` release anywhere, so seems fine to just delete the github tag/release and then update the version here

i just deleted the incorrect 2.0.1 releases/tags, so just bump this to 2.0.1 instead and then cut and release 2.0.1 again

Reply:
makes sense